cargo_build/x86_64

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:20.04
+
+RUN apt update && apt upgrade -y
+RUN apt install -y curl
+RUN curl https://sh.rustup.rs -sSf | sh -s -- --default-toolchain stable -y
+
+ENV PATH=$PATH:/root/.cargo/bin
+
+RUN rustup toolchain install nightly
+RUN rustup target add x86_64-unknown-linux-gnu
```

Comment:
This must be `rustup target add x86_64-unknown-linux-gnu --toolchain nightly` for this to work as expected.